softvis3d-frontend/package.json

```diff
@@ -14,7 +14,7 @@
     "prebuild:ci": "yarn run lint -s && yarn run test:coverage -s && yarn run test:sonarqube",
     "build:ci": "yarn run build:force -s",
     "build:force": "webpack --progress --colors --prod",
-    "lint": "tslint \"src/**/*.ts?(x)\" && shx echo \"[tslint] Lint successful\"",
+    "lint": "tslint \"src/**/*.ts?(x)\" && tslint \"test/**/*.ts?(x)\" && shx echo \"[tslint] Lint successful\"",
```

Comment:
Hm... Then the tests are supposed to be ready and debug free, whenever we execute them; even in development mode. I think running the linter with the `build`-step alone should suffice.

I'll just switch the computer and post updated scripts json i have in mind. 🏃